src/models/release.ts

```diff
@@ -0,0 +1,177 @@
+import { ReleaseType, ReleaseVoiced } from '../utils/types/releaseHelperTypes';
+import { VisualNovel } from './visualNovel';
+
+/**
+ * Represents producer involved into release.
+ */
+export interface ReleaseProducer {
+
+  /**
+   * Producer id.
+   */
+  id: number;
+
+  /**
+   * Is developer.
+   */
+  developer: boolean;
+
+  /**
+   * Is publisher.
+   */
+  publisher: boolean;
+
+  /**
+   * Producer name(romaji).
+   */
+  name: string;
+
+  /**
+   * Name in original.
+   */
+  originalName: string | null;
+
+  /**
+   * Type.
+   */
+  type: string;
+}
+
+/**
+ * Represents release media.
+ */
+export interface ReleaseMedia {
+
+  /**
+   * Medium.
+   */
+  medium: string;
```

Comment:
Medium?

Reply:
There is no info provided by official documentation.